picoquic/picoquic.h

```diff
@@ -598,10 +598,9 @@ void * picoquic_get_callback_context(picoquic_cnx_t* cnx);
 
 /* Send extra frames */
 int picoquic_queue_misc_frame(picoquic_cnx_t* cnx, const uint8_t* bytes, size_t length);
-int picoquic_queue_datagram_frame(picoquic_cnx_t* cnx, uint64_t id, size_t length, uint8_t* bytes);
 
 /* Send datagram frame */
-int picoquic_queue_datagram_frame(picoquic_cnx_t* cnx, uint64_t id, size_t length, uint8_t* bytes);
+int picoquic_queue_datagram_frame(picoquic_cnx_t* cnx, size_t length, const uint8_t* bytes);
```

Comment:
I like this kind of improvements by adding more "constness" 👍

Reply:
Yes. The "siduck" app is a bit of a joke, but it is a good way to shake up the "datagram" path.